helix-core/src/main/java/org/apache/helix/controller/stages/AttributeName.java

```diff
@@ -50,5 +50,11 @@
   // This attribute should only be used in TaskGarbageCollectionStage, misuse could cause race conditions.
   JOBS_WITHOUT_CONFIG,
   // This attribute should only be used in TaskGarbageCollectionStage, misuse could cause race conditions.
-  TO_BE_PURGED_JOBS_MAP
+  TO_BE_PURGED_JOBS_MAP,
+
+  // This attribute denotes the messages output from Per Preplica Throttle stage
+  PER_REPLICA_OUTPUT_MESSAGES,
+
+  // This attribute denotes the targeted partition state mapping from Per Preplica Throttle stage
+  PER_REPLICA_RETRACED_STATES
```

Comment:
I thought it has been commented on, so I did not comment in the previous review.
The names of these 2 fields are very misleading. Please try to make it understandable even without reading the comment.

Also please check the other namings in the PR. Although I don't like to be strict on the naming, we shall avoid confusing names. Please double check.

Reply:
You mean "PER_REPLICA_THROTTLED_OUTPUT_MESSAGES"? The previous review comment about another enum. thanks for pointing this out. Just checked, these two are the only left two.